plugins/modules/cloud/alicloud/ali_instance.py

```diff
@@ -505,19 +557,18 @@
         security_groups:
             description: One or more security groups for the instance.
             returned: always
-            type: list
-            elements: dict
+            type: complex
```

Comment:
This change looks wrong. Don't use `type: complex` except for special circumstances.

What exactly is `security_groups` returning?